### ISSUE TITLE
Document our process for communicating changes to the site

### ISSUE THREAD

**Issue by jeremyzilar** [GSA/digitalgov.gov#125]

**Comment by AndreaSigz** [comment]
Take a look at what I drafted @jeremyzilar 

Document our Process for Communicating Site Changes

Changes may be proposed in either GitHub as an issue, internally via a Google doc, from our shared email inbox, or social media. Once we’ve decided to make the changes and the updates have been made, several things (depending on the level of importance) should be done to alert our audience and stakeholders to these updates.

For Minor Changes, we should:

Create a Release Notes (or some other named) page to document the changes with:
the rationale behind the change, 
any research into the reasoning behind the change, if necessary, and 
any benefits the user will notice as a result of the change
link to the Issue in GitHub (if applicable) for more information, including a simplified way to follow along with the project in GitHub.
Announce a summary of the change in the internal Slack channel of our choice (#digitalgov, #OPP, #Federalist, etc), including why TTS’ers should care about this update 
Submit a summary of update to the GSA Today internal weekly newsletter 


For Major Changes, we should:

Create a Release Notes (or some other named) page to document the changes with:
the rationale behind the change, 
any research into the reasoning behind the change, if necessary, and 
any benefits the user will notice as a result of the change
link to the Issue in GitHub (if applicable) for more information, including a simplified way to follow along with the project in GitHub.
Write a blog, linking to the specific Release Notes entry (See #1 above), with any additional context to the problem or interesting information from GitHub
Add the blog, or link to Release Notes, to our newsletter
Tweet and add to Facebook, about the blog  
Announce a summary of the change in the internal Slack channel of our choice (#digitalgov, #OPP, #Federalist, etc), including why TTS’ers should care about this update 
Submit a summary of update to the GSA Today internal weekly newsletter 
Draft a note to the government-wide Communities of Practice, asking for their input and how they can comment/give feedback

------------------------------------------------------------------------------------------------------

(Optional) for Major Changes, we could:


Create a video showing the changes we made and the behind-the-scenes work undertaken
Follow with a survey to our federal audience about the change, inviting feedback, and follow-up with a subsequent blog about the feedback
